src/en/news/blog/2023/v17-2-6-quincy-released/index.md

```diff
@@ -0,0 +1,1038 @@
+---
+title: "v17.2.6 Quincy released"
+date: "2023-04-07"
+author: "yuriw"
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the sixth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- `ceph mgr dump` command now outputs `last_failure_osd_epoch` and
+  `active_clients` fields at the top level.  Previously, these fields were
+  output under `always_on_modules` field.
+
```

Comment:
@yuriw sorry for the last addition, can you add this line under "Notable changes"?
```suggestion
telemetry: Added new metrics to the ‘basic’ channel to report  per-pool bluestore compression metrics. See a sample report with `ceph telemetry preview`. Opt-in with `ceph telemetry on`.
```